requirements.txt

```diff
@@ -1 +1,3 @@
 click
+nose
```

Comment:
strictly speaking, nose is not a requirement for `nf-core/tools`, but one of many modules for executing tests. I think we do not have to list it here, but only in the .travis.yaml for CI.

Reply:
Get your point. Okay